tmpl/consent-banner.php

```diff
@@ -22,8 +34,25 @@
 				apply_filters( 'altis.consent.no_option_saved_message', $no_option_saved_message )
 			);
 		} else {
-			load_template( __DIR__ . '/cookie-preferences.php' );
-			load_template( __DIR__ . '/button-row.php' );
+			if ( 'none' !== $banner_option ) {
+				/**
+				 * Allow the cookie preferences template path to be overridden so it can be customized independently.
+				 *
+				 * @var string The path to the cookie preferences template.
+				 */
+				$cookie_preferences_template_path( 'altis.consent.cookie_preferences_template_path', __DIR__ . '/cookie-preferences.php' );
```

Comment:
It looks like this is missing an `apply_filters()` call.

Reply:
It sure is! 🤦‍♂️